io.dbeaver.DBeaverCommunity.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>io.dbeaver.DBeaverCommunity.desktop</id>
```

Comment:
The ID [should not](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-filespec) have the `.desktop` segment, it causes trouble – at least to me; that goes instead into the `<launchable type="desktop-id" />` tag that you are missing :wink:

And boy, your editor's formatting rules are all borked! :ghost:

Reply:
This entire thing got frankenstiend mostly from VSCode flathub entry, someone should do cleanup there ;)